cmd/main.go

```diff
@@ -1,31 +1,60 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"fun-with-go/fun"
 	"os"
 	"strconv"
 )
 
 func main() {
+
+	args := os.Args
+
 	//var number int
-	number, err := strconv.Atoi(os.Args[1])
+	number, err := strconv.Atoi(args[1])
```

Comment:
This is not right. Should be Args

Reply:
I see the assignment. Redundant it is, but this will work. Was seeing the code on phone earlier.